### PR TITLE
feat(locale): extend Hebrew locale data

### PR DESCRIPTION
**Pull request by shai-shimon**
### Added More

1. Colors
2. Music

### CODE REVIEW

**Comment by ST-DDT** [comment]
Please fix the lint errors.

File: src/locales/he/music/genre.ts [hunk 1]
```diff
@@ -17,4 +17,6 @@ export default [
   'בלוז',
   'לא מוסיקה',
   'ראפ',
+  'טראנס',
+  'האוס'
```
Comment (ST-DDT) [inline src/locales/he/music/genre.ts]:
```suggestion
  'האוס',
```

**Review by ST-DDT — APPROVED** [review]

File: src/locales/he/color/human.ts [hunk 2]
```diff
@@ -28,4 +28,10 @@ export default [
   'צבע תכלת',
   'לבנדר',
   'כסף',
+  'בורדו',
+  'בז',
+  'חאקי',
+  'ברונזה',
+  'ארד',
```
Comment (xDivisionByZerox) [inline src/locales/he/color/human.ts:35]: I get "bronze" for both of these values when translating them. Is that correct? Is this a duplicate?

Comment (ST-DDT) [reply src/locales/he/color/human.ts:35]: @shai-benshimol : ^

Comment (ST-DDT) [reply src/locales/he/color/human.ts:35]: @shai-benshimol : Please answer the question above.

Comment (matthewmayer) [reply src/locales/he/color/human.ts:35]: i checked with my friend who reads Hebrew, both are valid and could be used to describe colors, the first one is an approximation of the English word Bronze and is pronounced "bronza", the second one is the Hebrew word for the metal and is pronounced "arad".

**Review by matthewmayer — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**